cli/util/convert.go

```diff
@@ -222,15 +212,13 @@ func NewCommands() []*cli.Command {
 				},
 				{
 					Name:      "audit-bin",
-					Usage:     "Audit NeoFS container for duplicating block or index file objects",
-					UsageText: "neo-go util audit-bin -fs-rpc-endpoint <address1>[,<address2>[...]] --container <cid> [--block-attribute Block] [--index-attribute Index] [--wallet <wallet>] [--wallet-config <config>] [--address <address>] [--searchers <num>] [--retries <num>] [--debug] [--dry-run]",
+					Usage:     "Audit NeoFS container for duplicate and missing block objects",
+					UsageText: "neo-go util audit-bin -fs-rpc-endpoint <address1>[,<address2>[...]] --container <cid> [--block-attribute Block] [--wallet <wallet>] [--wallet-config <config>] [--address <address>] [--retries <num>] [--debug] [--dry-run]",
 					Action:    auditBin,
 					Flags:     auditBinFlags,
-					Description: `Searches for the duplicating index files or blocks. Logs duplicating
-   index files OIDs without removal if --dry-run is enabled. If --debug is enabled, then duplicating
-   block OIDs are also printed. Preserves index files consistency: the first index file returned by
-   SEARCH for the given height considered to be the original one, all duplicating blocks that are not
-   mentioned in this file will be removed.
+					Description: `Searches for the duplicating blocks. Logs duplicating and missing
```

Comment:
And it not only does searching, it also uploads/removes, add a note about this fact.